File: atcoder/answer.js
const fs = require('fs');

// Read the input from a file
// FIXME:change to '/dev/stdin' when submitting to atcoder
const input = fs.readFileSync('input.txt', 'utf8').split('\n');

let count = 0;

for (let i = 0 ; i < input.length ; i ++ ) {
    // index (count from 0)のときインデックスとおなじものがあったらカウント
    if (input[i].length == i + 1){
        count ++
    }
}

console.log(count)

File: atcoder/answer.py
import sys

# Redirect standard input to read from a file
sys.stdin = open('input.txt', 'r')

#########code from here here --------
def main():
    N = int(input())
    department = list(map(int, input().split()))
    department.sort()

    half = round(sum(department) / 2)

    group = 0
    for i in range(N) :
        if group < half :
            group = group + department[i]
            print(group)
main()